src/validations/app.js

```diff
@@ -0,0 +1,10 @@
+import * as Yup from 'yup';
+
+const appSchema = (t) => Yup.object().shape({
```

Comment:
Any other way to access t here?

Reply:
I can not use useTranslate() here, it's a hook so can be called inside React function component or custom React hook function
Or I can write validations right in validationSchema  and not import it

Reply:
We were talking about using i18n instance directly